.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Names are in alphabetical order, no priority or seniority is implied
+
+# repo owners
+*                                       @thekaveman @evansiroky 
```

Comment:
May I suggest `@cal-itp/gtfs-calitp-org-admins` instead? I just created this GitHub team, will be easier to manage without having to update this CODEOWNERS file anymore.